### PR TITLE
update perf run to the new mixer config

### PR DESCRIPTION
**Pull request by ldemailly**
```release-note
NONE
```

### CODE REVIEW

**Comment by douglas-reid** [comment]
are we postponing merging of this until `--useAst` is not required?

**Comment by ldemailly** [comment]
No I want to merge without waiting

File: devel/setup_run [hunk 1]
```diff
@@ -1,12 +1,15 @@
...
+mkdir -p emptydir
+mkdir -p mixerconfig
+cp mixer/testdata/config/* mixerconfig/
+rm mixerconfig/stackdriver.yaml
 cd mixer; set +x; source bin/use_bazel_go.sh ; set -x; cd ..
 ./istio/bazel-bin/devel/fortio/cmd/fortio/fortio server &
 ( cd proxy/src/envoy/mixer; ./start_envoy > /tmp/envoy.log ) &
 # add -v=5 for verbose/debug
-./mixer/bazel-bin/cmd/server/mixs server --configStoreURL=fs://$(pwd)/mixer/testdata/configroot --logtostderr 2> /tmp/mixs.2.log &
+./mixer/bazel-bin/cmd/server/mixs server --configStore2URL=fs://$(pwd)/mixerconfig --configStoreURL=fs://$(pwd)/emptydir --logtostderr 2> /tmp/mixs.2.log &
```
Comment (kyessenov) [inline devel/setup_run]: is default level 0?

Comment (ldemailly) [reply devel/setup_run]: yes afaik, those logs aren't too floody (modulo logging bugs that keep creeping up)

**Comment by ldemailly** [comment]
rerunning now to see if it still works

**Comment by ldemailly** [comment]
getting
```
[libprotobuf ERROR external/mixerclient_git/src/quota_cache.cc:116] Quota response did not have quota for: RequestCount
```
with this setup

**Comment by ldemailly** [comment]
@ZackButcher / @rshriram 
this PR isn't related to main line (so no risk) but is needed to run the benchmark - please approve

**Pull request merged**